### ISSUE TITLE
[Accessibility][Screen Reader, create bot dialog]: screen reader does not confirm once the option is choosed

### ISSUE THREAD

**Issue by liweitian** [microsoft/BotFramework-Composer#2684]
1. go to home page
2. click new
3. select create from scratch or create from template option, but the screen reader does not announce it.
